llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -758,6 +758,19 @@ void AsmPrinter::emitGlobalVariable(const GlobalVariable *GV) {
   // sections and expected to be contiguous (e.g. ObjC metadata).
   const Align Alignment = getGVAlignment(GV, DL);
 
+  // Identify globals with "asan_instrumented" attribute and extract
+  // the actual global variable size.
+  uint64_t ActualSize = 0;
+  if (GV->hasAttribute(Attribute::SanitizeAddress)) {
```

Comment:
This would be replaced by "asan_instrumented" attributes, once #68865 gets approved and merged.